site/docs/main/customize-installation.md

```diff
@@ -95,10 +97,76 @@ velero client config set features=
 
 ## Customize resource requests and limits
 
-By default, the Velero deployment requests 500m CPU, 128Mi memory and sets a limit of 1000m CPU, 256Mi.
-Default requests and limits are not set for the restic pods as CPU/Memory usage can depend heavily on the size of volumes being backed up.
+At installation, Velero sets default resource requests and limits for the Velero pod and the restic pod, if you using the [restic integration](/docs/main/restic/).
 
-Customization of these resource requests and limits may be performed using the [velero install][6] CLI command.
+<table caption="Velero Customize resource requests and limits defaults" >
+  <tr><th>Setting</th><th>Velero pod defaults</th><th>restic pod defaults (Velero 1.4.2 and later)</th></tr>
+  <tr><td>CPU request</td><td>500m</td><td>500m</td></tr>
+  <tr><td>Memory requests</td><td>128Mi</td><td>512Mi</td></tr>
+  <tr><td>CPU limit</td><td>1000m (1 CPU)</td><td>1000m (1 CPU)</td></tr>
+  <tr><td>Memory limit</td><td>256Mi</td><td>1024Mi</td></tr>
+</table>
+
+### Install with custom resource requests and limits
+
+You can customize these resource requests and limit when you first install using the [velero install][6] CLI command.
+
+```
+velero install \
+  --velero-pod-cpu-request <CPU_REQUEST> \
+  --velero-pod-mem-request <MEMORY_REQUEST> \
+  --velero-pod-cpu-limit <CPU_LIMIT> \
+  --velero-pod-mem-limit <MEMORY_LIMIT> \
+  [--use-restic] \
+  [--default-volumes-to-restic] \
+  [--restic-pod-cpu-request <CPU_REQUEST>] \
+  [--restic-pod-mem-request <MEMORY_REQUEST>] \
+  [--restic-pod-cpu-limit <CPU_LIMIT>] \
+  [--restic-pod-mem-limit <MEMORY_LIMIT>]
+```
+
+### Update resource requests and limits after install
+
+After installation you can adjust the resource requests and limits in the Velero Deployment spec or restic DeamonSet spec, if you are using the restic integration.
+
+**Velero pod**
+
+Update the `spec.template.spec.containers.resources.limits` and `spec.template.spec.containers.resources.requests` values in the Velero deployment.
+
+```bash
+kubectl patch deployment velero -n velero --patch \
+'{"spec":{"template":{"spec":{"containers":[{"name": "velero", "resources": {"limits":{"cpu": "1", "memory": "128Mi"}, "requests": {"cpu": "1", "memory": "256Mi"}}}]}}}}'
+```
+
+**restic pod**
+
+Update the `spec.template.spec.containers.resources.limits` and `spec.template.spec.containers.resources.requests` values in the restic DeamonSet spec.
+
+```bash
+kubectl patch daemonset restic -n velero --patch \
+'{"spec":{"template":{"spec":{"containers":[{"name": "restic", "resources": {"limits":{"cpu": "1", "memory": "512Mi"}, "requests": {"cpu": "1", "memory": "1024Mi"}}}]}}}}'
```

Comment:
Requests must be less than limits, so these values on memory should be swapped. Otherwise, the command works.

Reply:
whoops, this should be fixed now!